src/Directory.Build.props

```diff
@@ -6,14 +6,13 @@
     <Version>4.3.0</Version>
     <AnalysisLevel>5</AnalysisLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <NoWarn>CS1591</NoWarn>
```

Comment:
XML comments are now required for public members of Shouldy and Shouldy.DiffEngnie